android/src/main/java/io/openmobilemaps/mapscore/map/view/MapView.kt

```diff
@@ -50,6 +52,9 @@ open class MapView @JvmOverloads constructor(context: Context, attrs: AttributeS
 	private var saveFrameSpec: SaveFrameSpec? = null
 	private var saveFrameCallback: SaveFrameCallback? = null
 
+	private var mapViewStateMutableStateFlow = MutableStateFlow(MapViewState.UNINITIALIZED)
+	val mapViewStateStateFlow = mapViewStateMutableStateFlow.asStateFlow()
```

Comment:
Our usual naming convention is to have the MutableStateFlow have the `Mutable` suffix but not have the entire `StateFlow` suffix on both variables. So the non-mutable StateFlow would just be called `mapViewState` here